### PR TITLE
vendor dependencies

### PR DESCRIPTION
**Pull request by JAORMX**
This adds all the changes required to vendor our dependencies. It's pased on @tnozicka's https://github.com/openshift/file-integrity-operator/pull/14 PR which introduces this in the first place, but this adds the missing gosec dependency and changes to the makefile to make this work.

### CODE REVIEW

**Comment by JAORMX** [comment]
etcd call timed out...

**Comment by JAORMX** [comment]
rebased on top of the PR that introduced an extended timeout

File: go.mod [hunk 1]
```diff
@@ -4,16 +4,9 @@ go 1.13
 
 require (
 	github.com/go-openapi/spec v0.19.0
-	github.com/onsi/ginkgo v1.10.3 // indirect
-	github.com/onsi/gomega v1.7.1 // indirect
```
Comment (mrogers950) [inline go.mod]: Why are these removed? I think that if you run `go mod tidy` and then don't run `make e2e` you miss out on these deps. If you run `make e2e` at least once and then vendor, these will get included.

Comment (JAORMX) [reply go.mod]: I don't find where the usage of ginkgo is.
